user/api.go

```diff
@@ -312,6 +312,16 @@ func (a *Api) CreateCustodialUser(res http.ResponseWriter, req *http.Request, va
 		return
 	} else {
 		permissions := clients.Permissions{"custodian": clients.Allowed, "view": clients.Allowed, "upload": clients.Allowed}
+		if custodianUserID != "" {
```

Comment:
I don't think this block is required. The `care_partner` permission is used only by the care partner app. We should discuss this ticket.